Scout/ResponseJSONObjectSerializable.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ResponseJSONObjectSerializable.swift
+//  Scout
+//
+//  Created by Johnny Pham on 4/15/16.
+//  Copyright © 2016 Team Nato. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public protocol ResponseJSONObjectSerializable {
```

Comment:
That extension let us handle a response from Alamofire that returned an object as JSON and
create a Swift object (as long as the Swift class had the right kind of initializer, as per the
ResponseJSONObjectSerializable protocol). Now we need something similar but for a whole array of objects: We want to take a JSON array and turn it into an array of Swift objects. We’ll keep our protocol so we need to add to our project. Add a new ResponseJSONObjectSerializable.swift file and add the protocol to it.